crates/sui-types/src/crypto.rs

```diff
@@ -7,6 +7,7 @@ use std::str::FromStr;
 use anyhow::Error;
 use base64ct::Encoding;
 use digest::Digest;
+use narwhal_crypto::bls12381::{BLS12381Signature, BLS12381PublicKey, BLS12381KeyPair};
```

Comment:
Although I understand you probably need it for being able to test against another scheme, we should add support (and any mentioning) for BLS in another PR.

Reply:
Got it, I'll make sure any traces are gone before merging.

Reply:
Also, should we add Ecdsa in this PR?